AutofacContrib.NSubstitute.Tests/ExampleFixture.cs

```diff
@@ -175,6 +184,39 @@ public void Example_test_with_substitute_for_concrete()
             Assert.That(result, Is.EqualTo(val3));
         }
 
+        [Test]
+        public void SubstituteForAndProvide()
+        {
+            const int val = 2;
+
+            using var utoSubstitute = AutoSubstitute.Configure()
+                .SubstituteFor<ConcreteClass>(val).Provide(out var concreteClass).Configured()
+                .Build()
+                .Container;
+
+            Assert.NotNull(concreteClass.Configure());

```

Comment:
Also confused about the call here to `.Configure()` - shouldn't `concreteClass` be an instance of `ConcreteClassWithObject`, which doesn't have a `Configure` method? I'm sure I'm just missing something :)

Reply:
This is a check to make sure it's the NSubstitute version. The call to `.Configure()` fails if it isn't, and return a non-null value if it is.  I can switch that to a more descriptive testing extension